txpipe/masks.py

```diff
@@ -22,7 +21,7 @@ def run(self):
             bright_obj = f.read_map("bright_objects/count")
             depth = f.read_map("depth/depth")
             pixel_scheme = choose_pixelization(**metadata)
-
+            
```

Comment:
The linter is going to complain about this line. Either remove or remove the blank spaces on the left.

Reply:
Removed!